exploration/text-vs-code.md

```diff
@@ -85,14 +120,22 @@ according to its plural category
 So, in American English, the formatter might need to choose between formatting
 `You have 1 kilometer to go` and `You have 2 kilometers to go`.
 
-Rarely, messages needs to include leading or trailing whitespace due to
-e.g. how they will be concatenated with other text,
+Rarely do messages that need to include leading or trailing whitespace do so due to
+how they will be concatenated with other text,
```

Comment:
I think this change made the text far less clear. 

To be honest, I would have guessed that you would have replaced the somewhat biased word "Rarely" with something more neutral such as "Some messages need..."